api/v1/controllers/UserController.js

```diff
@@ -90,7 +90,8 @@ function requestPasswordReset(req, res, next) {
 router.use(bodyParser.json());
 router.use(middleware.auth);
 
-router.post('/', middleware.request(requests.BasicAuthRequest), createUser);
+router.post('/', middleware.request(requests.BasicAuthRequest),
+  middleware.permission(roles.ORGANIZERS), createUser);
```

Comment:
Does this mean that only authenticated organizers can create a user?

Reply:
Yeah, since we're reserving username/pass authentication for staff, mentors etc.. the assumption is that an authenticated staff member will create this type of account.

Reply:
So they create a user with a username and password (presumably for someone else). How is that different than the accredited endpoint? Just that we won't assign them a role initially?

Reply:
yeah